mne/io/eeglab/eeglab.py

```diff
@@ -178,9 +203,13 @@ def read_epochs_eeglab(input_fname, events=None, event_id=None, montage=None,
         If None, sensor locations are (0,0,0). See the documentation of
         :func:`mne.channels.read_montage` for more information.
     eog : list | tuple | 'auto'
-        Names or indices of channels that should be designated
-        EOG channels. If 'auto', the channel names containing
-        ``EOG`` or ``EYE`` are used. Defaults to empty tuple.
+        Names or indices of channels that should be designated EOG channels.
+        If 'auto', the channel names containing ``EOG`` or ``EYE`` are used.
+        Defaults to empty tuple.
+    misc : list or tuple
+        Names of channels or list of indices that should be designated MISC
+        channels. Values should correspond to the electrodes in the .set file.
+        Default is ``()``.
```

Comment:
Indeed, but as discussed previously, this is the precise wording I took from BV. We could have a new issue for this particular thing to standardize it across all readers?